Removed Trademark declaration from README.md

ElbowBaggins opened the pull request:
Custom Credit Systems no longer exists, and their acquirers most certainly did not renew this trademark. (http://tmsearch.uspto.gov/bin/showfield?f=doc&state=4810:xigqz4.2.1)

I'll bet you did not expect to see a pull request today.

ElbowBaggins committed:
Removed Trademark declaration from README.md

Custom Credit Systems no longer exists, and their acquirers most certainly did not renew this trademark. (http://tmsearch.uspto.gov/bin/showfield?f=doc&state=4810:xigqz4.2.1)

tedchoward commented:
I definitely did not ever expect a PR on this project.

So the trademark expired? I guess we should remove all the `(R)` suffixes after `ThinWire` in the source code headers as well. 🤷‍♂ 

I'll go ahead and merge this.

Thanks.

ElbowBaggins commented:
Hey, I only noticed this merge recently.

I actually work for the company that acquired CCS so I've wound up getting rather unusually familiar with The Toolkit Formerly Known as ThinWire. They've neglected to renew the trademark (we call it something else internally) so I made this PR as a bit of a goof.

We've converted "The Framework", as we call it, to a shadow DOM element and converted it to Typescript.

It's been quite a trip lol
